src/realm/object-store/sync/generic_network_transport.hpp

```diff
@@ -256,6 +256,8 @@ struct GenericNetworkTransport {
     }
 };
 
+const char* httpmethod_to_string(HttpMethod method);
```

Comment:
Why did this move here?

Reply:
It was originally in a test file and moved here so it could be used in app.cpp.